scripts/tcs_startup.sh

```diff
@@ -14,30 +14,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+KV_STORAGE="$TCF_HOME/tools/build/_dev/bin/kv_storage"
 enclave_manager="${TCF_HOME}/examples/enclave_manager/tcf_enclave_manager/enclave_manager.py"
 listener="${TCF_HOME}/common/python/connectors/direct/tcs_listener/tcs_listener.py"
 version="$(cat ${TCF_HOME}/VERSION)"
+# Default values
+DEFAULT_LISTENER_PORT=1947 # set in config/tcs_config.toml
+START_KV_STORAGE=1 # default if -s not passed
```

Comment:
Where is "-s" parameter is checked in the script? By default START_KV_STORAGE is set 1 and it is starting kv_storage service in tcf docker container as well.

Reply:
Looks fine to me. "-s" option is passed with tcs_startup.sh in docker compose file. So KV storage won't start.

Reply:
Another option is to remove the "-s" option from this script and start kv storage separately in stand alone build

Reply:
@Ram-srini: The `-s` option is parsed by `getopts` around line 75.
The `-s` option is passed from docker com a Dockerfile as mentioned by @manojgop.

@manojgop Yes, `kv_storage` can be started separately and that will work. But the purpose of `tcs_startup.sh` is to simplify things and start all the components.

Reply:
Got it. Thanks.